src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -187,10 +189,12 @@
 short-summary: Remove a network rule.
 examples:
   - name: Remove a network rule. (autogenerated)
-    text: az storage account network-rule remove --account-name MyAccount --resource-group MyResourceGroup --subnet mysubnet
+    text: |
+        az storage account network-rule remove --account-name MyAccount --resource-group MyResourceGroup --subnet mysubnet
```

Comment:
when users want to remove subnet in network rules, the correct usage is [--subnet ID | --subnet NAME --vnet-name NAME].

Reply:
Here just apply linebreak logic on examples. So I think the example is incorrect before this modification. I guess Aladdin generated a ambiguous value here. right?

Reply:
Updated manually